### DISCUSSION TITLE
Chat ideas and errors

### DISCUSSION THREAD

**Discussion by fsan2val** [owncast/owncast#3867] [General Owncast discussion]
Hello again.

1. I have tested the transmission chat on an Android mobile but when adding an emoticon, the window goes out of bounds and takes a while to process. If you close it and open it again, the window freezes, causing it to open and close whenever it wants. It even stops the video and says error and other things. I don't know if it happens only to me but I'll tell you what happened to me. (see video)

[https://drive.google.com/file/d/1hC16fhDxWpwKz96d8AQWllfFHusVQBmW/view?usp=sharing](url)

3. There is the possibility of fixing the chat so that it does not cover the video and remains below the video, YouTube style, that way we can see the video without fading and chat at the same time, it is a suggestion (See image)

![Screenshot_20240810-095808](https://github.com/user-attachments/assets/fade9411-dc6a-4be4-a50a-34236c26a09d)

Greetings good weekend

**Comment by gabek** [comment]
> There is the possibility of fixing the chat so that it does not cover the video and remains below the video, YouTube style, that way we can see the video without fading and chat at the same time

I appreciate your thoughts, but there are reasons things are like they are. It's not that we are dumb and didn't think of it. It's because there were so many problems of how it was previously, it had to be changed to how it is now. #3037.

There's always a story behind things, it's not just as easy as saying "do it like YouTube".

**Reply by fsan2val** [reply]
I understand everything. Thank you very much for the clarification, as always attentive. Regarding the emoticons, I don't know if it also happens to the rest of the community or if it's my phone, which when I open the emoticon window overflows, you can see it in the video. greetings to all

**Comment by gabek** [comment]
It's very possible it's happening to others, but I've never heard of it personally. There's a todo to replace the emoji picker, so when that happens it can be looked at. #3598

**Reply by fsan2val** [reply]
Thank you very much for responding and clarifying these concerns for me. kind regards
